common/src/main/java/com/wynntils/core/persisted/upfixers/config/CustomBankQuickJumpsUpfixer.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.persisted.upfixers.config;
+
+import com.wynntils.core.persisted.upfixers.RenamedPrefixesUpfixer;
+import com.wynntils.utils.type.Pair;
+import java.util.List;
+
+public class CustomBankQuickJumpsUpfixer extends RenamedPrefixesUpfixer {
```

Comment:
Checking `MapToMainMapRenamedConfigsUpfixer`, I think you can use `customBankPagesFeature.` -> `customBankQuickJumpsFeature` if only the feature name changed.

Reply:
That was where I looked to see how upfixing worked, but it didn't seem to work for me, just deleted the configs. Unless I'm blind and am missing an obvious mistake in this

`private static final List<Pair<String, String>> RENAMED_PREFIXES =
            List.of(Pair.of("customBankPagesFeature.", "customBankQuickJumpsFeature."));`

Reply:
It should work? Maybe try debugging it.

Reply:
I got it, didn't remove it from the applied upfixers in config so it wouldn't upfix it again and just deleted the invalid configs